### PR TITLE
Update getSimilarActionInQueue.js

### PR DESCRIPTION
**Pull request by nicemanis**
Implemented correct comparison of thunk action equality by comparing also the arguments of the functions. #163

### CODE REVIEW

**Comment by nicemanis** [comment]
I could add them in the near future, probably next week or week after that.

**Comment by d3vhound** [comment]
@nicemanis I'm still seeing only 1 thunk in the actionQueue even though the args are different.

File: src/utils/getSimilarActionInQueue.js [hunk 1]
```diff
@@ -12,12 +12,15 @@ export default function getSimilarActionInQueue(
...
-    return actionQueue.find((queued: *) => isEqual(queued, action));
+    return actionQueue.find((queued: *) => _.isEqual(queued, action));
   }
   if (typeof action === 'function') {
-    return actionQueue.find(
-      (queued: *) => action.toString() === queued.toString(),
-    );
+    return actionQueue.find((queued: *) => {
+      const isArgsEqual = _(action.meta.args)
+        .xorWith(queued.meta.args, _.isEqual)
```
Comment (chinesedfan) [inline src/utils/getSimilarActionInQueue.js]: Will comparing `args` one by one better? I think `_.xorWith` will treat `[1, 2]` with `[2, 1]` the same. What's more, with #208, users can customize their own `comparisonFn`.

**Comment by rgommezz** [comment]
This PR is not needed anymore. You can now [customise how the offline queue handles duplicates](https://github.com/rgommezz/react-native-offline#managing-duplicate-actions).

**Pull request closed without merging**